### PR TITLE
CORE-447 REPL++

### PR DESCRIPTION
**Pull request by rabbitonweb**
## Overview
Adds features like:
1. coloring
2. history
3. completion

### Does this PR relate to an RChain JIRA issue? 
CORE-447

### CODE REVIEW

File: node/src/main/scala/coop/rchain/node/repl.scala [hunk 1]
```diff
@@ -1,34 +1,47 @@
...
 
-    """
+object ReplRuntime {
+  def keywords: List[String] = List(
+    "stdOut",
+    "stdOutAck",
+    "stdErr",
+    "stdErrAck",
+    "for",
+    "!!"
```
Comment (goral09) [inline node/src/main/scala/coop/rchain/node/repl.scala]: What about the other keywords ?

**Comment by goral09** [comment]
@rabbitonweb this is approved. can you merge it?

File: node/src/main/scala/coop/rchain/node/repl.scala (hunk 1 above)
Comment (goral09) [reply node/src/main/scala/coop/rchain/node/repl.scala]: sent you a link long long time ago: https://github.com/rchain/rchain/blob/dev/rholang/src/main/bnfc/rholang_mercury.cf ;)

**Comment by rabbitonweb** [comment]
@goral09 conflicts ahead, this has low priority on my list, but i will close that by monday

**Pull request merged**